LICENSE

```diff
@@ -259,7 +259,7 @@ SOURCE/JAVA-CORE
       * core/src/main/java/org/apache/druid/java/util/common/concurrent/DirectExecutorService.java
 
     This product contains modified versions of the Dockerfile and related configuration files
-     from SequenceIQ's Hadoop Docker image, copyright SequenceIQ, Inc. (https://github.com/sequenceiq/hadoop-docker/)
+     from SequenceIQ's Hadoop Docker file, copyright SequenceIQ, Inc. (https://github.com/sequenceiq/hadoop-docker/)
```

Comment:
Suggest "This product contains modified versions of the Dockerfile, scripts, and related configuration files used for building SequenceIQ's Hadoop Docker image..."

Reply:
Thanks, fixed.